test/e2e/operator_test.go

```diff
@@ -883,6 +869,52 @@ func TestIngressGroup(t *testing.T) {
 	}
 }
 
+func ensureCredentialsRequest() error {
```

Comment:
```suggestion
func ensureCredentialsRequest() error {
    t.Helper()
```

Reply:
`Helper()` makes sense only in the functions which do something with the test: fail it, log something.  The helper functions should have `t *testing.T` as one of the parameters.

However, I found other functions which should be helpers: `createTestWorkload` and `waitForHTTPClientCondition` (from `util.go`).